Sources/Operation/RevealOperation.swift

```diff
@@ -9,21 +9,29 @@ import Foundation
  * operations.
  */
 public class RevealOperation: AbstractOperation {
+  /** The public key for the address being revealed. */
+  public let publicKey: String
+
   public override var dictionaryRepresentation: [String: String] {
     var operation = super.dictionaryRepresentation
     operation["public_key"] = publicKey
     return operation
   }
 
-  /** The public key for the address being revealed. */
-  public let publicKey: String
+  public override var defaultFees: OperationFees {
+    let fee = TezosBalance(balance: 0.001269)
+    let storageLimit = TezosBalance.zeroBalance
+    let gasLimit = TezosBalance(balance: 0.010000)
+    return OperationFees(fee: fee, gasLimit: gasLimit, storageLimit: storageLimit)
+  }
 
   /**
    * Initialize a new reveal operation for the given wallet.
    *
    * @param wallet The wallet that will be revealed.
+   * @param operationFees OperationFees for the transaction. If nil, default fees are used.
    */
-  public convenience init(from wallet: Wallet) {
+  public convenience init(from wallet: Wallet, operationFees _: OperationFees? = nil) {
```

Comment:
remove the unused operationFees argument?

Reply:
Should be passed through. Done.